lambdas/pii-redact/src/pii-redact-handler.ts

```diff
@@ -92,12 +93,18 @@ export class PiiRedactHandler implements LambdaInterface {
     if (!(await this.logStreamExists(logStreamName))) {
       logger.info("Creating log stream " + logStreamName);
 
-      await cloudwatch.send(
-        new CreateLogStreamCommand({
-          logGroupName: logGroupName,
-          logStreamName: logStreamName,
-        })
-      );
+      try {
+        await cloudwatch.send(
+          new CreateLogStreamCommand({
+            logStreamName: logStreamName,
+            logGroupName: logGroupName,
+          })
+        );
+      } catch (error: unknown) {
+        if (error instanceof ResourceAlreadyExistsException) {
+          logger.info(logStreamName + " already exists");
+        }
```

Comment:
let's add an else clause to cover the unexpected scenario. otherwise, looks good - just make sure to update the tests